test/test_symmetry.jl

```diff
@@ -154,21 +154,22 @@ end
         k = 6
         i = 1
         cryst = Sunny.diamond_crystal()
+        O = Sunny.StevensMatrices(N)
 
         # print_site(cryst, i)
-        Λ = 𝒪[6,0]-21𝒪[6,4]
+        Λ = O[6,0]-21O[6,4]
         @test Sunny.is_anisotropy_valid(cryst, i, Λ)
 
         R = [normalize([1 1 -2]); normalize([-1 1 0]); normalize([1 1 1])]
         # print_site(cryst, i; R)
-        Λ = 𝒪[6,0]-(35/√8)*𝒪[6,3]+(77/8)*𝒪[6,6]
+        Λ = O[6,0]-(35/√8)*O[6,3]+(77/8)*O[6,6]
         Λ′ = rotate_operator(Λ, R)
         @test Sunny.is_anisotropy_valid(cryst, i, Λ′)
 
         latvecs = lattice_vectors(1.0, 1.1, 1.0, 90, 90, 90)
         cryst = Crystal(latvecs, [[0., 0., 0.]])
         # print_site(cryst, i)
-        Λ = randn()*(𝒪[6,0]-21𝒪[6,4]) + randn()*(𝒪[6,2]+(16/5)*𝒪[6,4]+(11/5)*𝒪[6,6])
+        Λ = randn()*(O[6,0]-21O[6,4]) + randn()*(O[6,2]+(16/5)*O[6,4]+(11/5)*O[6,6])
```

Comment:
Using `O` here (instead of `𝒪`) doesn't look great when juxtaposed with coefficients. The tradeoff is that some people's terminals won't render `𝒪` properly, so I was thinking it might be better to use plain ASCII in the main examples.

Reply:
Probably, users will make themselves clear by inserting `*` for clarity. Also consider that this form of juxtaposed coefficients is a special feature of julia, so people are already wary. This term: `(O[6,2]+(16/5)*O[6,4]+(11/5)*O[6,6])` looks fine to me, and even more fine with spaces `(O[6,2] + (16/5)*O[6,4] + (11/5)*O[6,6])`.

Definitely Unicode-optional is the way to go.